components/camera/replaypcd/replaypcd.go

```diff
@@ -224,6 +317,12 @@ func (replay *pcdCamera) Reconfigure(ctx context.Context, deps resource.Dependen
 		}
 	}
 
+	if replayCamConfig.BatchSize == nil {
+		replay.limit = 1
+	} else {
+		replay.limit = *replayCamConfig.BatchSize
```

Comment:
[q] Should we call it batch size in both places, rather than BatchSize in one and limit in the other?

Reply:
Got it, if we're following patterns established elsewhere, this seems fine!